Updated configuring realms chapter #1483 (#1533)
keycloak/keycloak-documentation: server_admin/topics/realms/proc-creating-a-realm.adoc

. Point to the top of the left pane.

ifeval::[{project_product}==true]
. Click *Add Realm*.
endif::[]
ifeval::[{project_community}==true]
. Click *Create Realm*.
endif::[]
+
.Create realm menu

> image:{project_images}/create-realm-menu.png[Create realm menu]

Reviewer: This images is missing when I build the SSO version.  To fix it, you can rename add-realm-menu.png as create-realm-menu.png.

.Create realm
image:{project_images}/create-realm.png[Create realm]

ifeval::[{project_product}==true]
The current realm is now set to the realm you just created.  You can switch between managing different realms by pointing to the top left corner to click *Select Realm*.
endif::[]
ifeval::[{project_community}==true]

> The current realm is now set to the realm you just created. You can switch between managing different realms by pointing to the top left corner to select from dropdown.

Reviewer: ```suggestion
The current realm is now set to the realm you just created. You can switch between realms by clicking the realm name in the menu.
```
